project/src/main/scala/licenses/report/HTMLWriter.scala

```diff
@@ -0,0 +1,268 @@
+package src.main.scala.licenses.report
+
+import java.io.{BufferedWriter, PrintWriter}
+import java.nio.file.Files
+
+import sbt.File
+
+/**
+  * Allows to create very simple HTML reports.
+  *
+  * @param bufferedWriter writer to which the generated code will be written
+  */
+class HTMLWriter(bufferedWriter: BufferedWriter) {
+  val writer = new PrintWriter(bufferedWriter)
+
+  /**
+    * Writes the header containing basic styles and scripts for the reports.
+    *
+    * @param title the page title
+    */
+  def writeHeader(title: String): Unit = {
+    val heading =
+      s"""<html>
+         |<head>
+         |<meta charset="utf-8">
+         |<script src="https://code.jquery.com/jquery-1.12.4.js"></script>
+         |<script src="https://code.jquery.com/ui/1.12.1/jquery-ui.js"></script>
+         |<title>$title</title>
+         |<style>
+         |table, th, td {
+         | border: solid 1px;
+         |}
+         |h4 {
+         |  font-weight: normal;
+         |  display: inline;
+         |}
+         |</style>
+         | <script>
+         |$$( function() {
+         |  $$( ".accordion" ).accordion({
+         |    active: false,
+         |    collapsible: true
+         |  });
+         |});
+         |</script>
+         |</head>
+         |<body>""".stripMargin
```

Comment:
I do not see a simple way of doing it better. I considered using some HTML-generation library but I think it may be an overkill for this usecase, especially as it is an internal tool.